### DISCUSSION TITLE
recommend

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#168] [Announcements]
# recommend

VitePress相关的主题，博客，插件汇总推荐

https://theme.sugarat.top/recommend.html

**Comment by colin404** [comment]
1111

**Comment by colin404** [comment]
22222

**Reply by ATQQ** [reply]
3333

**Reply by Justin3go** [reply]
44444

**Reply by haskio** [reply]
55555

**Comment by haskio** [comment]
有没有1.0版本增加百度统计、google统计之类的差距或者方法。

**Reply by ATQQ** [reply]
可以呀这个是官方 VitePress 提供了入口的，通过 [head](https://vitepress.dev/reference/site-config#head)  配置。
```ts
export default defineConfig({
   head:[
        [
          'script',
          {
           // 设置需要的 script 标签属性
            src: 'sdk  地址',
          }
        ],
        [
          'script',
          {},
          'xxxx' // 设置 script 标签里的内容
        ],
      ]
})
```

@haskio

**Comment by xkfe** [comment]
有没有像 element 组件库那样的组件预览插件推荐呀？可以预览，复制，在 github 打开按钮的那种

**Reply by ATQQ** [reply]
应该有，有不少组件库用 VitePress 搭的应该。

比如这个 嘿嘿

https://ikun-ui.netlify.app/components/KButton.html
